client/dist/style.css

```diff
@@ -1,10 +1,106 @@
+.title {
+  font-size: 40px;
+  display: flex;
+  flex-wrap: wrap;
+  font-family: Circular,-apple-system,BlinkMacSystemFont,Roboto,Helvetica Neue,sans-serif;
+}
+
+p {
+  margin-bottom: 5px;
+  font-family: Circular,-apple-system,BlinkMacSystemFont,Roboto,Helvetica Neue,sans-serif;
+  font-size: 16px;
+}
+
 body {
```

Comment:
@SimonC989 @carl12 Note that everyone's component will ultimately end up in the same body tag, subject to these styles. Best to avoid styling global elements (same goes for the <p> styling above)